packages/preact-iso/prerender.js

```diff
@@ -2,13 +2,23 @@ import { h, options, cloneElement } from 'preact';
 import renderToString from 'preact-render-to-string';
 
 let vnodeHook;
+let initialized = false;
 
 const old = options.vnode;
 options.vnode = vnode => {
 	if (old) old(vnode);
 	if (vnodeHook) vnodeHook(vnode);
 };
 
+async function init() {
+	const fs = (await eval('u=>import(u)')('fs')).promises;
+	// eslint-disable-next-line no-undef
+	globalThis.fetch = async url => {
+		const text = () => fs.readFile('dist/' + String(url).replace(/^\//, ''), 'utf-8');
```

Comment:
We can't assume the user has `dist/` as their output directory. Will need to pass the value of `--out`.